etc/WMAgentConfig.py

```diff
@@ -228,7 +228,6 @@
 config.ErrorHandler.namespace = "WMComponent.ErrorHandler.ErrorHandler"
 config.ErrorHandler.componentDir = config.General.workDir + "/ErrorHandler"
 config.ErrorHandler.logLevel = globalLogLevel
-config.ErrorHandler.maxRetries = maxJobRetries
```

Comment:
@goughes can you please add this line back? I _think_ we need it for the T0 case.